docs/start/install.md

```diff
@@ -203,7 +213,7 @@ inv server
 For more server options, run:
 
 ```
-inv server -h
+inv server -a
```

Comment:
`-h` is the correct option to show help menu. Why the change here?

Reply:
Yes you are right. I misinterpreted the command wrong. I was looking for the option to start the server with a diffrent IP. I'm not very experianced, sorry.

Reply:
No worries :)